python/_typing.py

```diff
@@ -0,0 +1,3 @@
+from typing import Any, Tuple
+
+CAN_MSG = Tuple[int, Any, bytes, int]
```

Comment:
not necessarily required, just wasn't sure where to put this. seems better to have a name assigned to it rather than inlining `Tuple[int, Any, bytes, int]` everywhere.